tq/basic_download.go

```diff
@@ -88,10 +88,12 @@ func (a *basicDownloadAdapter) download(t *Transfer, cb ProgressCallback, authOk
 		defer dlFile.Close()
 	}
 
-	rel, err := t.Actions.Get("download")
+	rel, err := t.Rel("download")
 	if err != nil {
 		return err
-		// return errors.New("Object not found on the server.")
+	}
+	if rel == nil {
+		return errors.Errorf("Object %s not found on the server.", t.Oid[:7])
```

Comment:
Good catch! I just pushed up https://github.com/git-lfs/git-lfs/pull/2004/commits/2ff28591073fc8f04cb235c5d3a57228a7be6a79 to address. 👍